src/rp2_common/hardware_irq/irq.c

```diff
@@ -135,9 +135,9 @@ irq_handler_t irq_get_exclusive_handler(uint num) {
 static uint16_t make_branch(uint16_t *from, void *to) {
     uint32_t ui_from = (uint32_t)from;
```

Comment:
or cast straight to signed int here?
does the calculation of delta change if ui_from and ui_to are signed? eyeballing i would say no.